editor/editor_help.cpp

```diff
@@ -3084,3 +3086,9 @@ void FindBar::_search_text_submitted(const String &p_text) {
 		search_next();
 	}
 }
+
+void FindBar::_focus_lost() {
+	if (Input::get_singleton()->is_action_pressed(SNAME("ui_cancel"))) {
+		_hide_bar();
+	}
+}
```

Comment:
```suggestion
}

```
Missing line termination

Reply:
Thanks!